exercises/book-store/src/example/java/Bookstore.java

```diff
@@ -0,0 +1,64 @@
+import java.awt.print.Printable;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Bookstore{
+
+	private int bookPrice, maxGroupSize;
+	private double discountIncrement;
+	private List<Integer> books;
+	private static double[] discount_tiers = {0,5,10,20,25};   	
```

Comment:
I added `static` to constants `BOOK_PRICE`, `MAX_GROUP_SIZE`.
Also, I deleted the `discountIncrement` because it's useless now.